src/lib/server/constants.ts

```diff
@@ -1,6 +1,6 @@
 import { constants } from '@clerk/backend/internal';
-import { env as privateEnv } from '$env/dynamic/private';
-import { env as publicEnv } from '$env/dynamic/public';
+import * as privateEnv from "$env/static/private"
+import * as publicEnv from "$env/static/public"
```

Comment:
Can you format this for me please? Thanks

Reply:
I’ve updated the import statements as requested and formatted the code accordingly.
Please let me know if any further changes are needed. Thanks